### ISSUE TITLE
In chinese,伍 is another five

### ISSUE THREAD

**Issue by liuyanhui** [jackdcrawford/five#192]
In chinese,伍 is another five. It is used in Money.
In chinese, all the numbers have two styles.
 1,  2,  3,  4,  5,  6,  7,  8,  9, 10
一, 二, 三, 四, 五, 六, 七 ,八, 九, 十
壹, 贰, 叁, 肆, 伍, 陆, 柒, 捌, 玖, 拾

The third style is  used in paper money and coin.

**Comment by KID-WuMeng** [comment]
Should divide the Simplified Chinese and Traditional Chinese.

**Comment by nanowang** [comment]
@KID-WuMeng 
They're the same character in simplified and traditional Chinese.

**Comment by undoZen** [comment]
It's like lower case and upper case in English, but only numbers have those two representation.

**Comment by mordrax** [comment]
Man.. those traditionalists must have had alot of time... one (壹) has 11 strokes! Traditional Binary must have sucked big time!

**Comment by CrazyPython** [comment]
No, it's cardinal vs. ordinal. Fifth vs. Five. Not so extraordinary. Get over it.
